app-frontend/src/app/pages/admin/organization/organization.html

```diff
@@ -1,10 +1,13 @@
 <div class="container-admin-header">
   <div class="admin-header">
     <div class="admin-logo image-placeholder editable-image"
```

Comment:
I would remove the `editable-image` class here so that whenever it is fetching (no matter if it has logo uri or not, or if the user is admin/super user or not), a gray avatar will show up.